wizeline/remix-project-lab
# Feat - 64 - Project Resources

# What does this PR do?

Adds section to manage project resources

# Where should the reviewer start?

app/routes/projects/$projectId/index.tsx
app/models/project.server.ts
app/models/resource.server.ts

# How should this be manually tested?

- Go to a project page and try the new Resources section
- Check that it can add, update and delete resources

# Any background context you want to provide?



# What are the relevant tickets?

[Issue 64](https://github.com/wizeline/remix-project-lab/issues/64)

# Screenshots



# Questions



# Checklist
